plugin/mouse/mouse_scroll.py

```diff
@@ -181,13 +206,17 @@ def mouse_scroll_continuous(new_scroll_dir: Literal[-1, 1]):
         continuous_scroll_mode = "scroll down continuous"
```

Comment:
As written, I believe this will always display "scroll down continuous"

Reply:
I fixed that.